cerberus/tests/test_validation.py

```diff
@@ -1270,8 +1297,20 @@ def test_oneof_schema():
 
 
 def test_nested_oneof_type():
-    schema = {'nested_oneof_type':
-              {'valueschema': {'oneof_type': ['string', 'integer']}}}
+    schema = \
```

Comment:
put it back (I change it as I had a problem with the flake8 validation...but now it is working fine)